examples/roses/package.json

```diff
@@ -40,6 +40,7 @@
     "@material/react-switch": "^0.5.0",
     "@material/react-text-field": "^0.5.1",
     "@material/react-top-app-bar": "^0.5.0",
+    "@material/react-typography": "0.0.0",
```

Comment:
is this right??

Reply:
lerna hasn't updated. So as of right now it is.